### PR TITLE
Add Teamwork.com github action

### PR DESCRIPTION
**Pull request by miguelbemartin**
This PR adds the action to keep the PRs in sync with Teamwork.com tasks.

### CODE REVIEW

**Comment by sdras** [comment]
Please title case the link as per CONTRIBUTING.md. Thanks!

File: README.md [hunk 1]
```diff
@@ -359,6 +359,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Create an Outlook Calendar Event using Microsoft Graph](https://github.com/anoopt/ms-graph-create-event)
 - [Watch for GitHub Wiki page changes and post to Slack](https://github.com/benmatselby/gollum-page-watcher-action)
 - [Send an SMS using MessageBird](https://github.com/nikitasavinov/messagebird-sms-action)
+- [Keep your PRs in sync with Teamwork tasks](https://github.com/miguelbemartin/teamwork-github)
```
Comment (miguelbemartin) [inline README.md:362]:
```suggestion
- [Keep Your PRs in Sync With Teamwork Tasks](https://github.com/miguelbemartin/teamwork-github)
```

**Comment by miguelbemartin** [comment]
@sdras done!

**Comment by sdras** [comment]
Thank you!

**Comment by miguelbemartin** [comment]
Nice one! 👍

**Pull request merged**